airflow/api/common/experimental/get_dags.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from airflow.models import DagBag
+from airflow import configuration
+
+
+def get_dags(is_paused=None):
+    """
+    Returns a list of all Dags
+    :return: List of all DAGs
+    """
+    dagbag = DagBag()
```

Comment:
Constructing the dag bag involves parsing all the dags - this is not what we want to do. Instead we should look at the models.DAG.

Reply:
done.